### ISSUE TITLE
Forbidden response for certificate request

### ISSUE THREAD

**Issue by enesozturk** [planetscale/planetscale-node#31]
Hi, today I was working with Planetscale and `planetscale-node` package. I followed all the steps in [docs](https://docs.planetscale.com/tutorials/connect-nodejs-app). When I run a basic query example on Node.js, I get a forbidden response with a `403` status code. 

It happens in certificate request in [postJSON function](https://github.com/planetscale/planetscale-node/blob/b8d181f41b2bc59e6ebf701cfbbb939320a0d16d/src/index.ts#L129). I  debugged the code and got all the `header`, `body` data from the debugger, tried to send post request on **Postman** with same informations and I got 200. 

After a few hours, wanted to try `axios` for this request and it was successful again. Cannot sure the `http.request` implementation is wrong somehow or it is just me.

What do you think?

## Code
```js
const { PSDB } = require('./dist/index.js')
const conn = new PSDB('main')

async function main() {
  const [rows, fields] = await conn.execute('select * from likes')
  console.log(rows, fields)
}

main()

```

## Stack Trace

```bash
undefined:1
<html>
^

SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at IncomingMessage.<anonymous> (REPO_PATH/planetscale-node/dist/index.js:102:69)
    at IncomingMessage.emit (node:events:402:35)
    at endReadableNT (node:internal/streams/readable:1343:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

## Packages

- Node.js: v16.13.0
- planetscale-node: v0.2.0

**Comment by williamrijksen** [comment]
Same here!

**Comment by enesozturk** [comment]
Fixed the issue and will open pr with saveral improvements 🤙🏽

**Comment by williamrijksen** [comment]
Awesome @enesozturk !

**Comment by dgraham** [comment]
We narrowed this down to a missing `User-Agent` request header, which explains why tools like curl and postman worked while the JS request didn't. Give the fix a try in `planetscale-node@0.3.0`. Thanks for reporting this!
